messages/add.md

```diff
@@ -30,6 +30,14 @@ Specify a language
 
 Specify a path
 
+# output.successfullyAddedRules
```

Comment:
These messages were previously hardcoded in `RuleAddAction.ts`, so I moved them here.